Tests.Integration.Presentation.Web/External/BusinessTypesTest.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Core.DomainModel.ItSystem;
+using Core.DomainModel.Organization;
+using Presentation.Web.Models.External.V2;
+using Presentation.Web.Models.External.V2.Response;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Integration.Presentation.Web.Tools.External;
+using Tests.Toolkit.Patterns;
+using Xunit;
+
+namespace Tests.Integration.Presentation.Web.External
+{
+    public class BusinessTypesTest :WithAutoFixture
+    {
+        [Fact]
+        public async Task Can_Get_AvailableBusinessTypes()
+        {
+            //Arrange
+            var orgUuid = DatabaseAccess.MapFromEntitySet<Organization, Guid>(x => x.AsQueryable().Where(x => x.Id == TestEnvironment.DefaultOrganizationId).Select(x => x.Uuid).SingleOrDefault());
+            var pageSize = 5; //Minimum is 1;
+            var pageNumber = 0; //Always takes the first page;
+            var dbData = DatabaseAccess.MapFromEntitySet<BusinessType, IEnumerable<Guid>>(x => x.AsQueryable().OrderBy(x => x.Name).Where(x => x.IsObligatory).Select(x => x.Uuid).Take(pageSize).ToList());
```

Comment:
Test gerne at dem der ikke er obligatory men enabled i org også er med i dine expected. Ellers er denne test ikke robust over for andre tests